lib/project_types/script/messages/messages.rb

```diff
@@ -100,26 +99,26 @@ module Messages
           metadata_schema_versions_missing_minor: "Invalid Script API metadata:" \
                                                   " 'schemaVersions' is missing the 'minor' field",
 
-          metadata_not_found_cause: "Script version file (%s) cannot be found.",
-          metadata_not_found_help: "Ensure the 'shopify/scripts-toolchain-as' package is up to date and " \
+          metadata_not_found_cause: "Can't find the script version file (%s).",
+          metadata_not_found_help: "Make sure the 'shopify/scripts-toolchain-as' package is up to date and " \
                                      "'package.json' contains a 'scripts/build' entry with a " \
                                      "'--metadata build/metadata.json' argument",
```

Comment:
This suggestion is specific to TypeScript and doesn't apply to Wasm. It also won't apply to Rust scripts either. Should we generalize this error message?

Reply:
Yes. This should be generalized. Do you have a suggestion?

Reply:
Hmm this is hard because each language does its own thing currently (e.g. Wasm and TS handle the metadata differently). We have plans to have a single metadata file in the future, but that probably won't be for a while.

Maybe we can say something like: `Make sure your project is up-to-date and a script metadata file is accessible at {path}`. That second part might be redundant though.